src/Internal/Service/OrderService.php

```diff
@@ -254,6 +254,104 @@ public function attach_exchange_info_to_order( int $order_id, WC_Payments_API_Ch
 		$order->save_meta_data();
 	}
 
+	/**
+	 * Gets currently attached intent ID of the order.
+	 *
+	 * @param  int $order_id Order ID.
+	 *
+	 * @return string|null Intent ID for the order. Null if no intent ID attached to order.
+	 * @throws Order_Not_Found_Exception
+	 */
+	public function get_intent_id( int $order_id ): ?string {
+		$order     = $this->get_order( $order_id );
+		$intent_id = (string) $order->get_meta( '_intent_id', true );
+		if ( empty( $intent_id ) ) {
+			return null;
+		}
+		return $intent_id;
+	}
+
+	/**
+	 * Gets cart hash for the given order ID.
+	 *
+	 * @param  int $order_id  ID of the order.
```

Comment:
RadoCS:
```suggestion
	 * @param int $order_id  ID of the order.
```

Note: There are a few more spaces like this in the file, but I won't add comments for them.